docs/api-reference.md

```diff
@@ -171,6 +253,33 @@ curl -XGET localhost:10101/version
 {"version":"v0.6.0"}
 ```
 
+### Get status
+
+`GET /status`
+
+Returns the status of nodes in a Pilosa cluster.
```

Comment:
`/status` actually returns the status of the cluster. It includes the list of nodes in the cluster, but currently we don't provide a "node status".